standard_forms/generated/standard_forms.json

```diff
@@ -12,21 +12,21 @@
             "query.graphql"
           ],
           "convert_data": null,
-          "custom_wasm_function": null
+          "convert_data_type": "Extism"
```

Comment:
Are we not removing it extism for standard reports? 👀

Reply:
This is is tricky, we will actually keep this variant of [ConvertDataType](https://github.com/msupply-foundation/open-msupply/blob/3c37126fd0958d2d9b83affb539bdf6a4e8de22a/server/service/src/report/definition.rs#L82-L86), even when we remove Extism, and this data type Serializes by default to "Extism", there is no effect here though since, since convert_data above is null. Maybe we should change the default to BoaJs on that data file in develop, but it doesn't hurt having it as Extism either for now:  https://github.com/msupply-foundation/open-msupply/blob/41d19ccfbb05cf411de1d653cf88c8c0a2226ce8/server/service/src/report/report_service.rs#L545-L547

Reply:
Okiie should be fine, can you make an issue to default to BoAJs in develop~? Thanks, would be nice to have it default to the one we're actually using

Reply:
https://github.com/msupply-foundation/open-msupply/issues/7328